app/views/pages/manage_metadata.html.erb

```diff
@@ -1,182 +0,0 @@
-        <h3>Comments Received</h3>
-        <ul>
-          <li>
-            <div class="comments">
-              <p>2</p>
-            </div>
-            <div class="info">
-              <p><time datetime="2015-04-23">2014-04-23</time> | ASF57FR15<br>Global Cyclone Lorem Ipsum</p>
-            </div>
-          </li>
-          <li>
-            <div class="comments">
-              <p>2</p>
-            </div>
-            <div class="info">
-              <p><time datetime="2015-04-23">2014-04-23</time> | ASF57FR15<br>Global Cyclone Lorem Ipsum</p>
-            </div>
-          </li>
-          <li>
-            <div class="comments">
-              <p>2</p>
-            </div>
-            <div class="info">
-              <p><time datetime="2015-04-23">2014-04-23</time> | ASF57FR15<br>Global Cyclone Lorem Ipsum</p>
-            </div>
-          </li>
-          <li>
-            <div class="comments">
-              <p>2</p>
-            </div>
-            <div class="info">
-              <p><time datetime="2015-04-23">2014-04-23</time> | ASF57FR15<br>Global Cyclone Lorem Ipsum</p>
-            </div>
-          </li>
-          <li>
-            <div class="comments">
-              <p>2</p>
-            </div>
-            <div class="info">
-              <time datetime="2015-04-23">2014-04-23</time> | ASF57FR15
-              <p>Global Cyclone Lorem Ipsum</p>
-            </div>
-          </li>
-          <li>
-            <a href="#">More</a>
-          </li>
-        </ul>
-      </div>
-    </section> -->
-
-    <!-- <section class="callout-box col-left">
-      <div class="callout-content">
-        <h3>Your Recently Published Collections</h3>
-        <ul>
-          <li>
-            <p><time datetime="2015-04-23">2014-04-23</time> | ASF57FR15<br>Global Cyclone Lorem Ipsum</p>
-          </li>
-          <li>
-            <p><time datetime="2015-04-23">2014-04-23</time> | ASF57FR15<br>Global Cyclone Lorem Ipsum</p>
-          </li>
-          <li>
-            <p><time datetime="2015-04-23">2014-04-23</time> | ASF57FR15<br>Global Cyclone Lorem Ipsum</p>
-          </li>
-          <li>
-            <p><time datetime="2015-04-23">2014-04-23</time> | ASF57FR15<br>Global Cyclone Lorem Ipsum</p>
-          </li>
-          <li>
-            <p><time datetime="2015-04-23">2014-04-23</time> | ASF57FR15<br>Global Cyclone Lorem Ipsum</p>
-          </li>
-          <li>
-            <a href="#">More</a>
-          </li>
-        </ul>
-      </div>
-    </section> -->
-
-    <section class="eui-callout-box col-left">
-      <h3 class="eui-callout-box__title">Upload XML Files</h3>
-      <div class="eui-callout-box__list">
-        <a class="eui-btn eui-btn--sm disabled" href="javascript:void(0);">Select files</a><span class="coming-soon">Coming Soon</span>
-      </div>
-    </section>
-
-    <!-- <section class="callout-box col-left">
-      <div class="callout-content">
-        <h3>Collection Templates</h3>
```

Comment:
We may be working on Templates soon-ish right? Would it be helpful to keep this section?